src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -1428,7 +1428,7 @@ public C(string s)
     {
         if (string.IsNullOrWhiteSpace(s))
         {
-            throw new ArgumentException(""message"", nameof(s));
+            throw new ArgumentException($""'{nameof(s)}' cannot be null or empty"", nameof(s));
```

Comment:
Shouldn't this be cannot be null or whitespace?

Reply:
nope

Reply:
Why not?

Reply:
I do not know in what context this was called exception

Reply:
i think @YairHalberstadt  is simply saying that the string should be `$""'{nameof(s)}' cannot be null or whitespace""`  since that's the check that is actually happening.